os/binfmt/libelf/libelf_load.c

```diff
@@ -96,6 +96,27 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#ifdef CONFIG_APP_BINARY_SEPARATION
+/****************************************************************************
+ * Name : get_mpu_region_size
+ *
+ * Description:
+ *   Return a size value which is a power of two as per MPU restrictions
+ *
+ * Returned value:
+ *   Size which is power of two
+ ***************************************************************************/
+static size_t get_mpu_region_size(size_t sz)
+{
+	int i;
+	for (i = 5; i < 32; i++) {
```

Comment:
Limits could be added to defines rather than hard coding

Reply:
Done.